Sources/ClientRuntime/Networking/AwsChunked/AWSChunkedStream.swift

```diff
@@ -0,0 +1,159 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+
+/// Reads data from the input stream, chunks it, and passes it out through its output stream.
+class AWSChunkedStream {
```

Comment:
This used to be a protocol, I changed it to a concrete stream type that chunks its input and outputs an AWS-chunked stream.